CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 [Full changelog](https://github.com/mozilla-rally/core-addon/compare/v0.9.0...master)
 
+* [#392](https://github.com/mozilla-rally/rally-core-addon/pull/392): Implement the new consent logic to prevent running side-loaded studies.
```

Comment:
Nice job catching this issue btw :)

Reply:
To be honest, I didn't! @knowtheory did :-)